sequence_processing_pipeline/BCLConvertJob.py

```diff
@@ -0,0 +1,162 @@
+        # w/environment variables is to do the same on QSUB but with -v instead of
+        # --export. The syntax of the values are otherwise the same.
+        # -v <variable[=value][,variable2=value2[,...]]>
+
+        # declare a name for this job to be sample_job
+        lines.append("#PBS -N %s" % self.job_name)
+
+        # what torque calls a queue, slurm calls a partition
+        # SBATCH -p SOMETHING -> PBS -q SOMETHING
+        # (Torque doesn't appear to have a quality of service (-q) option. so this
+        # will go unused in translation.)
+        lines.append("#PBS -q %s" % queue_name)
+
+        # request one node
+        # Slurm --ntasks-per-node=<count> -> -l ppn=<count>	in Torque
+        lines.append("#PBS -l nodes=%d:ppn=%d" % (node_count, nprocs))
+
+        # Slurm --export=ALL -> Torque's -V
+        lines.append("#PBS -V")
+
+        # Slurm walltime limit --time=24:00:00 -> Torque's -l walltime=<hh:mm:ss>
+        # using the larger value found in the two scripts (36 vs 24 hours)
+        lines.append("#PBS -l walltime=%d:00:00" % wall_time_limit)
+
+        # send email to charlie when a job starts and when it terminates or
+        # aborts. This is used to confirm the package's own reporting
+        # mechanism is reporting correctly.
+        lines.append("#PBS -m bea")
+
+        # specify your email address
+        # TODO: Send an email to jeff, and qiita.help as well.
+        lines.append("#PBS -M ccowart@ucsd.edu")
+
+        # min mem per CPU: --mem-per-cpu=<memory> -> -l pmem=<limit>
+        # taking the larger of both values (10G > 6G)
+        #lines.append("#PBS -l pmem=10gb")
+
+        # --output -> -o
+        lines.append("#PBS -o %s" % self.stdout_log_path)
+        lines.append("#PBS -e %s" % self.stderr_log_path)
+
+        # there is no equivalent for this in Torque, I believe
+        # --cpus-per-task 1
+        # By default, PBS scripts execute in your home directory, not the
+        # directory from which they were submitted. Use root_dir instead.
+        lines.append("set -x")
+        lines.append("date '+%s' > /{}/{}.log".format(self.root_dir, self.job_name))
+        lines.append("module load bclconvert_3.7.5")
+        lines.append("cd %s"  % self.root_dir)
+        # lines.append("export PATH=$PATH:/usr/local/bin")
+        lines.append('%s --sample-sheet %s --mask-short-adapter-reads \
+                      1 -R . -o %s --loading-threads 8 --processing-threads \
+                      --minimum-trimmed-read-length 1 \
+                      8 --writing-threads 2 --create-fastq-for-index-reads \
+                      --ignore-missing-bcls' % (self.bclconvert_path,
+                                                 self.sample_sheet_path,
+                                                 self.output_directory))
+        lines.append("echo $? >> /%s/%s.log" % (self.root_dir, self.job_name))
+        lines.append("date '+%s' >> /{}/{}.log".format(self.root_dir, self.job_name))
```

Comment:
Can you add a command copying the Stats file to the `/sequencing/complete_runs/run_id/...`